app/styles/app.scss

```diff
@@ -0,0 +1,233 @@
+
+$button-grey: #ddd;
+$button-green: #80cbc4;
+$header-bar-blue: #00bbd3;
+$title-white: #fff;
+$regular-text: rgba(255, 255, 255, 0.6);
+$placeholder-text: rgba(255, 255, 255, 0.4);
+$green-text: #92cbc5;
+$button-text-grey: #1c1c1c;
+$background-primary: #282828;
+
+
+$font-family: Roboto;
+
+$base-font-size: 16px;
+$base-line-height: 24px;
+
+
+body {
+  margin: 0;
+
+}
+
+a:link {
+  text-decoration: none;
+}
+
+main {
+margin: auto;
+max-width: 60rem;
+
+}
+
+nav {
+  background: $header-bar-blue;
+  color: $title-white;
+  overflow: auto;
+}
+
+.navbar-heading {
+  font-family: $font-family;
+  margin-left: 1rem;
+}
+
+.container {
+  background: $background-primary;
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  min-height: 100vh;
```

Comment:
`min-height` should be set on body (or whatever contains this container and navbar) and then the container would be flex 1 Similar to this sticky footer example (minus the footer markup): https://philipwalton.github.io/solved-by-flexbox/demos/sticky-footer/